keep-ui/app/alerts/quality/page.tsx

```diff
@@ -0,0 +1,12 @@
+import QualityAlert from './alert-quality-table'; // Adjust the import path if needed
+
+const AlertQualityPage = () => {
```

Comment:
I guess we don't need this page anymore, right?

Reply:
Page is not needed. but component is needed

Reply:
resolved